### PR TITLE
Database Week2 is ready for comments

### PR DESCRIPTION
**Pull request by aijazraja**

### CODE REVIEW

File: databases/week2/week2_part1.sql [hunk 1]
```diff
@@ -0,0 +1,30 @@
+-- Add a task with these attributes: title, description, created, updated, due_date, status_id, user_id
+insert into task(title, description, created, updated, due_date, status_id, user_id)
+values('meeeting','meeting with comapany owner','2021-01-15 task','2021-01-19 06:54:16','2021-02-18 06:54:16',2,3);
```
Comment (pritalpatel) [inline databases/week2/week2_part1.sql]:
```suggestion
values('meeeting','meeting with comapany owner','**2021-01-15 task**','2021-01-19 06:54:16','2021-02-18 06:54:16',2,3);
```
I think 'created' as a DATE datatype, Its not allow to insert string.

File: databases/week2/week2_part2.sql [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+name varchar(45) not null,
+email varchar(25),
+phone int(15),
+class_id int(10)not null,
+primary key(id),
+foreign key (class_id) references class(id)
+);
+-- Create an index on the name column of the student table.
+create index index_name
+on Student(name);
```
Comment (pritalpatel) [inline databases/week2/week2_part2.sql]: Give a proper name when create a index for ex. index_student
```suggestion
create index index_student
on Student(id);
```
index on a VARCHAR column will boost query performance
It's make more sense to find a INT column to create INDEX on it.

File: databases/week2/week2_part3.sql [hunk 3]
```diff
@@ -0,0 +1,41 @@
+/*
```
Comment (aina21) [inline databases/week2/week2_part3.sql:1]: you should use data for lesson2 for this part.
https://github.com/HackYourFuture-CPH/databases/blob/master/lesson2/lesson2-data.sql

**Review by aina21 — CHANGES_REQUESTED** [review]
good job

File: databases/week2/week2_part4.sql [hunk 4]
```diff
@@ -0,0 +1,56 @@
+
+-- -----------------------------------------------------
+-- Table Movies
+-- -----------------------------------------------------
+CREATE TABLE Movies (
+  id INT NOT NULL AUTO_INCREMENT,
```
Comment (pritalpatel) [inline databases/week2/week2_part4.sql]:
```suggestion
  id INT(5) NOT NULL AUTO_INCREMENT,
```
Define a specific range for the INT(n) data type.
It's consider on the performance, compression and indexes

File: databases/week2/week2_part1.sql (hunk 1 above)
Comment (aina21) [reply databases/week2/week2_part1.sql:3]: you can use NOW(),
https://www.w3schools.com/sql/func_mysql_now.asp

File: databases/week2/week2_part4.sql (hunk 4 above)
Comment (aijazraja) [reply databases/week2/week2_part4.sql]: Hi pritalpatel
Thanks for your feedback
i have fixed all of your proposed comments

**Pull request merged**